src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
Seems OK except for the file size check. If you're going that route you'll have to do a proper checksum comparison, not just size.